fastlane/actions/test_sample_code.rb

```diff
@@ -95,10 +95,8 @@ def self.is_supported?(platform)
       end
 
       def self.fill_in_env_variables
-        # Some code samples need a value in a certain env variable
-        ["GITHUB_TOKEN"].each do |current|
```

Comment:
Why not just leave the `each` and add the `CI` parameter here? It's nice to make it easy to add further parameters.